### PR TITLE
upload the project to review it

### PR DESCRIPTION
**Pull request by saied2035**
### **review the project:**:
 make a review for a project i made at microverse trials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# review-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (saied2035) [inline README.md:5]: README file should descripe the project to let the others know your project and how it work better. Please, check this link : [README template](https://github.com/microverseinc/readme-template)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+<<<<<<< HEAD
```
Comment (saied2035) [inline index.html:10]: HTML should only tags or comments if needed, make sure you delete any unneeded contents.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (saied2035) [inline index.html:54]: remove unneeded comments to make your project professional.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  <body>
+<<<<<<< HEAD
+    <nav id="navbar">
+      <a href="https://saied2035.github.io/robofriends-react/" target="_blank"><ul>robofriends</ul></a>
+      <a href="https://saied2035.github.io/smartbrain/" target="_blank"><ul>smartbrain</ul></a>
+    </nav>
+    <div id="welcome-section">
+      <h1>welcome to my portfolio</h1>
+      <p class="welcome">
+         Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
```
Comment (saied2035) [inline index.html:18]: your portfilio is a way to show yourself. make sure it's breif and descripe your projects

File: index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (saied2035) [inline index.html:5]: title should be related to the page contents.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
```
Comment (saied2035) [inline index.html:55]: this project don't have any javascript , please remove this line.

**Review by saied2035 — COMMENTED** [review]
### your project is a good one :
- using flexbox 
- giving proper classes or tags for the elements

but there are some issues you should resolve

## general issues:

1. you don't setup linters for your project. remember linters helps you make your project looks professional,fixes errors and suggests best practices.
2. when dividing HTML to sections make sure you use section tag not div
3. README file should descripe the project to let the others know your project and how it work better. Please, check this link : [README template](https://github.com/microverseinc/readme-template)
4. your project should contains the file it need please remove unneeded files like js folder 
5. see the inline comments

feel free to leave comments or communicate with me if anything is not 100% clear

**Review by ReshyResh — APPROVED** [review]
# Hi @saied2035 ,

### Great review! 
  - ✔Descriptive comments and reasons for the requested changes.
  - ✔Used Github's review function correctly.
  - ✔Requests are polite suggestions rather than orders.

Your project is complete! Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**